src/pyhf/readxml.py

```diff
@@ -366,8 +367,13 @@ def parse(configfile, rootdir, track_progress=False):
         ],
         'version': schema.version,
     }
-    schema.validate(result, 'workspace.json')
-
+    try:
+        schema.validate(result, 'workspace.json')
+    except exceptions.InvalidSpecification as exc:
+        if validation_as_error:
+            raise exc
```

Comment:
Hm. @kratsg do you know why this isn't getting covered?

Reply:
Oh, we don't have anything that covers reading an XML that's invalid.. I can add that pretty easily as another validation.

Reply:
Or just as a mock.